doc/source/reference/distutils.rst

```diff
@@ -1,3 +1,6 @@
+.. for doctests
+   >>> with open('foo.c', 'w') as f: pass
+
```

Comment:
Seems odd to leave this at the top while the others are at the bottom

Reply:
Fair enough, it will move :-)

Reply:
It is even more useful that the import and the creation of the file foo.c are necessary at the same place.